bin/download-cardano-db-sync-latest.sh

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env bash
+
+# TODO this is very similar to the kupo script, we might want to make this dry
```

Comment:
Do we want to create an issue? I see we also repeat the R2 endpoint multiple times in the codebase

Reply:
Yes, that makes sense but since there are other a lot of TODOs in the codebase, before the next milesone I'd like to go through them and decide if we want to create issues or not - and remove TODO comments in both the cases